Src/BootCamp.Chapter/ContactsCenter.cs

```diff
@@ -1,16 +1,35 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using System.Text;
 
 namespace BootCamp.Chapter
 {
     public class ContactsCenter
     {
-        private readonly List<Person> _people;
+        private readonly List<Person> _people = new List<Person>();
 
         public ContactsCenter(string peopleFile)
         {
-            // load people
+            string[] people;
+
+            try
+            {
+                people = File.ReadAllLines(peopleFile);
+            }
+            catch (Exception)
```

Comment:
No need to specify anything if you catch all.

Reply:
Removed (Exception) after catch.